src/Microsoft.DotNet.Build.Tasks/Microsoft.DotNet.Build.Tasks.csproj

```diff
@@ -108,13 +59,23 @@
     </PropertyGroup>
     <ItemGroup>
       <PackageFiles Include="$(MSBuildThisProjectDirectory)PackageFiles\**\*.*" />
-      <NativeBinariesx86 Include="$(PackagesDir)LibGit2Sharp.0.19.0.0\lib\net40\NativeBinaries\x86\*.*" />
-      <NativeBinariesamd64 Include="$(PackagesDir)LibGit2Sharp.0.19.0.0\lib\net40\NativeBinaries\amd64\*.*" />
       <NuProjFiles Include="$(PackagesDir)NuProj\$(NuProjVersion)\tools\**\*.*" />
     </ItemGroup>
+    <ItemGroup>
+      <PublishedRuntimeItems Include="@(ReferenceCopyLocalPaths)">
+        <DestinationRelativePath>%(FileName)%(Extension)</DestinationRelativePath>
+      </PublishedRuntimeItems>
+
+      <PublishedRuntimeItems Include="@(Content)">
+        <DestinationRelativePath>%(FileName)%(Extension)</DestinationRelativePath>
+      </PublishedRuntimeItems>
+
+      <PublishedRuntimeItems Include="@(None)">
```

Comment:
mm I checked the build and it didn't cause any conflicts, plus this will be eventually removed as well when we upgrade the build tools package since None is replaced by ReferenceCopyLocalPaths